core/store/store.go

```diff
@@ -134,18 +134,18 @@ func NewStore(config *orm.Config) *Store {
 func NewStoreWithDialer(config *orm.Config, dialer Dialer) *Store {
 	err := os.MkdirAll(config.RootDir(), os.FileMode(0700))
 	if err != nil {
-		logger.Fatal(fmt.Sprintf("Unable to create project root dir: %+v", err))
+		panic(fmt.Sprintf("Unable to create project root dir: %+v", err))
 	}
 	orm, err := initializeORM(config)
 	if err != nil {
-		logger.Fatal(fmt.Sprintf("Unable to initialize ORM: %+v", err))
+		panic(fmt.Sprintf("Unable to initialize ORM: %+v", err))
 	}
 	ethrpc, err := dialer.Dial(config.EthereumURL())
 	if err != nil {
-		logger.Fatal(fmt.Sprintf("Unable to dial ETH RPC port: %+v", err))
+		panic(fmt.Sprintf("Unable to dial ETH RPC port: %+v", err))
 	}
 	if err := orm.ClobberDiskKeyStoreWithDBKeys(config.KeysDir()); err != nil {
-		logger.Fatal(fmt.Sprintf("Unable to migrate key store to disk: %+v", err))
+		panic(fmt.Sprintf("Unable to migrate key store to disk: %+v", err))
```

Comment:
What's the reason for using panic over logger.Fatal? :thinking:

Reply:
I tend to prefer it from a stylistic/readability angle, since it looks more like a control statement than a function, but I'm happy to change it.

Reply:
Just wondering, but IMHO consensus on it is important otherwise its application is inconsistent and we risk changing between panic vs. fatal because of differing opinions

Reply:
I'm actually swayed by the argument that it looks more like a control statement, it also works a little better with go's detection of unused code. But I feel like the convention is to avoid panic in our code base in favour of using `logger`.